### PR TITLE
Fix #139. Split beneficial_owner_table

### LINKED ISSUE #139: Split BeneficialOwnership subclasses into their own Tables
BeneficialOwnership entity has two subclasses, PersonBeneficialOwner and OrganizationBeneficialOwner.

There is one table that has the union of the fields needed for this. We kept that as is for expediancy in converting over to H5, but single-table inheritance is not as clear as joined table inheritance, and is not consistent with Entity and its derivatives. 

Please update the table and change these classes.

Progress on #36 and comments from PR #133

### PR DESCRIPTION
**Pull request by frankduncan**
Issue #139: Split BeneficialOwnership subclasses into their own Tables

### CODE REVIEW

File: psm-app/db/changelog/db-changelog-1.0.xml [hunk 1]
```diff
@@ -1616,4 +1616,162 @@
...
+    <rollback>
+      <sql>
+        UPDATE beneficial_owner bo SET
+          person_ind = (SELECT 'Y' FROM person_beneficial_owners WHERE beneficial_owner_id = bo.beneficial_owner_id),
+          middle_name = (SELECT middle_name FROM person_beneficial_owners WHERE beneficial_owner_id = bo.beneficial_owner_id),
+          first_name = (SELECT first_name FROM person_beneficial_owners WHERE beneficial_owner_id = bo.beneficial_owner_id),
+          last_name = (SELECT last_name FROM person_beneficial_owners WHERE beneficial_owner_id = bo.beneficial_owner_id),
+          ssn = (SELECT ssn FROM person_beneficial_owners WHERE beneficial_owner_id = bo.beneficial_owner_id),
+          birth_dt = (SELECT birth_dt FROM person_beneficial_owners WHERE beneficial_owner_id = bo.beneficial_owner_id),
+          hired_at = (SELECT hiderd_at FROM person_beneficial_owners WHERE beneficial_owner_id = bo.beneficial_owner_id),
```
Comment (jasonaowen) [inline psm-app/db/changelog/db-changelog-1.0.xml]: Typo: `hiderd_at` should be `hired_at`.

File: psm-app/db/changelog/db-changelog-1.0.xml [hunk 2]
```diff
@@ -1616,4 +1616,162 @@
...
+          relationship_type_code = (SELECT relationship_type_code FROM beneficial_owner WHERE beneficial_owner_id = bo.beneficial_owner_id);
+        UPDATE beneficial_owner bo SET
+          person_ind = (SELECT 'N' FROM orgnization_beneficial_owners WHERE beneficial_owner_id = bo.beneficial_owner_id),
```
Comment (jasonaowen) [inline psm-app/db/changelog/db-changelog-1.0.xml]: Typo: `orgnization_beneficial_owners` should be `organization_beneficial_owners`.

File: psm-app/db/changelog/db-changelog-1.0.xml [hunk 3]
```diff
@@ -1616,4 +1616,162 @@
...
+          fein = (SELECT fein FROM organization_beneficial_owners WHERE beneficial_owner_id = bo.beneficial_owner_id),
+          legal_name = (SELECT legal_name FROM organization_beneficial_owners WHERE beneficial_owner_id = bo.beneficial_owner_id);
```
Comment (jasonaowen) [inline psm-app/db/changelog/db-changelog-1.0.xml]: Each of these update queries incorrectly update all rows. I think we need to tack on a `WHERE` clause at the end, like

```sql
WHERE EXISTS (
  SELECT *
  FROM organization_beneficial_owners obo
  WHERE obo.beneficial_owner_id = bo.beneficial_owner_id
)
```

(Adapted from [this StackOverflow answer](https://stackoverflow.com/a/1293347/25637).)

**Comment by frankduncan** [comment]
Jenkins, retest this please.

File: psm-app/db/changelog/db-changelog-1.0.xml [hunk 4]
```diff
@@ -1655,6 +1655,7 @@
     </rollback>
   </changeSet>
 
+<<<<<<< HEAD
```
Comment (jasonaowen) [inline psm-app/db/changelog/db-changelog-1.0.xml]: Whoops, we have a merge conflict marker left behind!

**Pull request merged**